offchain-modules/packages/x/src/multisig/multisig-mgr.ts

```diff
@@ -52,7 +62,17 @@ export interface ckbCollectSignaturesPayload {
   txSkeleton: TransactionSkeletonObject;
 }
 
-export type collectSignaturesParamsPayload = ethCollectSignaturesPayload | ckbCollectSignaturesPayload;
+export interface ckbUnlockCollectSignaturesPayload {
+  sigType: SigType;
+  unlockRecords?: unlockRecord[];
+  createAssets?: createAsset[];
```

Comment:
what is `createAssets` field for?

Reply:
refactored ckbCellCollectSignaturesPayload to mint, createCell, unlock different types, the createAssets field is used only when createCell, moved to ckbCreateCellCollectSignaturesPayload